simulation/navigator_gazebo/models/wamv/wamv.sdf

```diff
@@ -2,22 +2,24 @@
 <sdf version="1.5">
   <model name="wamv">
     <link name="base_link">
-      <!-- TODO: Tune these -->
+      <gravity>0</gravity>
```

Comment:
I tried to do the state setting two different ways, the first way required the boat to have 0 gravity but this new way doesn't.